src/utils/PrivateRoute.js

```diff
@@ -0,0 +1,21 @@
+import { useSelector } from 'react-redux';
+import { Route, Redirect } from 'react-router-dom';
+import { LOADING } from '../actions/constants';
+import '../assets/styles/components/PrivateRoute.scss';
+
+const PrivateRoute = ({ component: Component, ...rest }) => {
+  const authUser = useSelector((state) => state.auth_status);
+  console.log(authUser);
```

Comment:
Please remove console.log

Reply:
Hi Leonel, the console log is now removed.